README.md

```diff
@@ -0,0 +1,74 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Weather App
+
+> This is a weather app that allows you to search for a city and get the current weather information,
+Get all the information you need to plan your day. The app displays the current temperature, the weather description, the humidity, the wind speed, and the pressure.
+
+<br>
+<br>
+
+## Built With
+
+<br>
+<img alt="React Badge" src="https://img.shields.io/badge/React-20232A?style=for-the-badge&logo=react&logoColor=61DAFB">
+<img alt="Redux" src="https://img.shields.io/badge/Redux-593D88?style=for-the-badge&logo=redux&logoColor=white">
+<img alt="HTML Badge" src="https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white">
+<img alt="CSS Badge" src="https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white">
+<img alt="JavaScript Badge" src="https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E">
+<img alt="Jest Badge" src="https://img.shields.io/badge/Jest-C21325?style=for-the-badge&logo=jest&logoColor=white"><br>
+
+<br>
+<br>
+
+## Getting Started with Create React App and Redux
+
```

Comment:
- [ ] Nice Working on your project, kindly add a link to the live version of your project using Heroku, Netlify, or GitHub Pages.
- [ ] Kindly add a 5mins recorded video presenting the features of the project.